app/src/main/java/ar/com/wolox/android/example/BaseConfiguration.kt

```diff
@@ -0,0 +1,9 @@
+package ar.com.wolox.android.example
+
+internal open class BaseConfiguration {
```

Comment:
Asymmetric blank lines for this class.

EDIT: These seems to be happening for most classes. Do you see any value in correcting this?